server/lib/whispers/whisper-api.ts

```diff
@@ -68,6 +74,12 @@ export class WhisperApi {
     container.resolve(WhisperService)
   }
 
+  @httpGet('/sessions')
+  @httpBefore(throttleMiddleware(getWhisperSessionsThrottle, ctx => String(ctx.session!.user!.id)))
+  async getWhisperSessions(ctx: RouterContext): Promise<SbUserId[]> {
```

Comment:
I think we should have learned by this point that you should basically never return a raw array from API methods, always put it within an object so we can add new fields when necessary.

Additionally, this should almost certainly return the info for the users they have sessions with so that it doesn't have to be retrieved separately.